src/main/java/sorald/rule/RuleType.java

```diff
@@ -1,9 +1,20 @@
 package sorald.rule;
 
 /** Enumeration of Sonar rule types */
-public enum RuleType {
-    BUG,
-    VULNERABILITY,
-    CODE_SMELL,
-    SECURITY_HOTSPOT;
+public enum RuleType implements IRuleType {
+    BUG("Bug"),
+    VULNERABILITY("Vulnerability"),
+    CODE_SMELL("Code_Smell"),
+    SECURITY_HOTSPOT("Security_Hotspot");
```

Comment:
Any specific reason to pass an argument to the constructor? We can use `name` method of `Enum` to get the string representation which will default to `BUG` and so on.

Reply:
The name method always returns the name in uppercase and is not really changeable. Using a String would allow us to change the name(the String) in the future. Furthermore, printing these is way easier because you don't need a lot of String manipulations. For example, you would print `Security_Hotspot` more likely than `SECURITY_HOTSPOT`. Converting `SECURITY_HOTSPOT` to a user-friendly, readable version requires unnecessary complex logic.

Reply:
> The name method always returns the name in uppercase and is not really changeable.

Sounds good. 

> Furthermore, printing these is way easier because you don't need a lot of String manipulations

If it is supposed to be an output for user, it is better to name it `toString` than `getName`.